tracer/src/Datadog.Trace/Iast/Location.cs

```diff
@@ -28,18 +29,22 @@ public Location(string method)
         }
     }
 
-    public Location(string? stackFile, string? methodName, int? line, ulong? spanId, string? methodTypeName)
+    public Location(StackFrame? stackFrame, ulong? spanId)
     {
-        if (!string.IsNullOrEmpty(stackFile))
-        {
-            this.Path = System.IO.Path.GetFileName(stackFile!.Replace('/', System.IO.Path.DirectorySeparatorChar).Replace('\\', System.IO.Path.DirectorySeparatorChar));
-            this.Line = line;
-        }
-        else
-        {
-            this.Method = methodName;
-            this.Path = methodTypeName;
-        }
```

Comment:
We don't want the path name as well as the filename, if we have it?

Reply:
We don't have fields to store both, and we decided it's better to have always the fully qualified type name